client/packages/system/src/Stock/Components/Repack/RepackEditForm.tsx

```diff
@@ -6,42 +6,41 @@ import {
   NumericTextInput,
   TextWithLabelRow,
   useTranslation,
+  RepackNode,
 } from '@openmsupply-client/common';
 import {
   LocationRowFragment,
   Repack,
   StockLineFragment,
-  useStock,
 } from '@openmsupply-client/system';
 import { LocationSearchInput } from '../../..';
 import React, { FC, useEffect, useState } from 'react';
 
 const INPUT_WIDTH = 100;
 
 interface RepackEditFormProps {
-  invoiceId?: string;
   stockLine: StockLineFragment | null;
   onChange: (repack: Repack) => void;
   draft: Repack;
+  repackData?: RepackNode;
 }
 
 export const RepackEditForm: FC<RepackEditFormProps> = ({
-  invoiceId,
   onChange,
   stockLine,
   draft,
+  repackData,
```

Comment:
Seeing as this is just a form component, do you think we could only pass in a draft? And let the calling component decide whether the that draft is the `draft` from the edit hook or `repackData.from`?

Reply:
Yeah, good idea. Will do.

Reply:
b2c78a562 :)